icu4c/source/common/unicode/localpointer.h

```diff
@@ -162,11 +162,11 @@ class LocalPointerBase {
     T *ptr;
 private:
     // No comparison operators with other LocalPointerBases.
-    bool operator==(const LocalPointerBase<T> &other);
-    bool operator!=(const LocalPointerBase<T> &other);
+    bool operator==(const LocalPointerBase<T> &other) = delete;
```

Comment:
I think there's other code (not related to MF2) that also includes pragmas to disable warning 4661, for the same reason, that could be removed now that these operators are deleted. Could be a different PR, though.

Reply:
... and fixes for other code should be under a different ticket. Let's finish up ICU 75.

Reply:
Yes, those pragmas will be possible to delete now, but between the release candidate and the release I'd like to do only necessary changes.